Dockerfile

```diff
@@ -1,7 +1,11 @@
 FROM jboss/wildfly:8.2.0.Final
 MAINTAINER Democracy Works, Inc. <dev@democracy.works>
 
+ENV HOST_IP "TO.BE.FILLED.IN"
+
 COPY ./start /bin/start
 
+ADD standalone-rsyslog.xml /opt/jboss/wildfly/standalone/configuration/standalone-rsyslog.xml
+
 ENTRYPOINT ["/bin/start"]
-CMD ["-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
+CMD ["-c", "standalone-rsyslog.xml", "-Djboss.host.name=$HOST_IP", "-b", "0.0.0.0", "-bmanagement", "0.0.0.0"]
```

Comment:
I tried with the default value at first, I think it was either blank if unset or was some random value, I don't really recall exactly what since it was a couple of Fridays ago. If you want to show me how other containers are doing it, I'm happy to give it a try, I didn't like setting a faked up env var.